x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/parser/SqlParserTests.java

```diff
@@ -44,6 +45,11 @@ public void testSelectScore() {
         assertEquals("SCORE", f.functionName());
     }
 
+    public void testSelectLeft() {
```

Comment:
Minor thing: method called `testSelectLeft()` but the test itself is about the `RIGHT` function.